src/coordinator/index.spec.js

```diff
@@ -51,6 +51,111 @@ describe('Coordinator', function() {
     });
   });
 
+  describe('createFavoriteApplication', function() {
+    context('when the application ID is provided', function() {
+      let expectedApplicationFavorite;
+      let applicationFavoriteFromServer;
+      let promise;
+      let request;
+      let toCamelCase;
+
+      beforeEach(function() {
+        expectedApplicationFavorite = fixture.build(
+          'contxtUserFavoriteApplication'
+        );
+        applicationFavoriteFromServer = fixture.build(
+          'contxtUserFavoriteApplication',
+          expectedApplicationFavorite,
+          {
+            fromServer: true
+          }
+        );
+
+        request = {
+          ...baseRequest,
+          post: this.sandbox.stub().resolves(applicationFavoriteFromServer)
+        };
+        toCamelCase = this.sandbox
+          .stub(objectUtils, 'toCamelCase')
+          .callsFake((app) => expectedApplicationFavorite);
+
+        const coordinator = new Coordinator(baseSdk, request);
+        coordinator._baseUrl = expectedHost;
+        promise = coordinator.createFavoriteApplication(
+          expectedApplicationFavorite.applicationId
```

Comment:
This is fine but it feels a little weird since the `createFavoriteApplication` is to be called with the application id and not using the application favorite application id. Especially since you're doing it with the regular application in the DELETE [here](https://github.com/ndustrialio/contxt-sdk-js/pull/97/files#diff-b4dd03a165bf28f14cfa732bcf0c235eR128)

Maybe before line 63 you can do the 
```
application = fixture.build('contxtApplication');
```
and then have
```
expectedApplicationFavorite = fixture.build(
  'contxtUserFavoriteApplication',
  {  applicationId: application.id }
);
```
or something

Reply:
Maybe I'm just being weird. You don't have to make this change if you don't want to

Reply:
I had the exact same thought as I was using the application fixture for the DELETE test but forgot to go back and change it. 

I'll fix, thanks

Reply:
Fixed 👍